stubs/mu-plugin.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/*
+ * Plugin Name: CacheWerk k6 (MU)
+ * Plugin URI: https://github.com/cachewerk/k6
+ * Description: ...
+ * Version: 1.0.0
+ * Author: Rhubarb Group
+ * Author URI: https://rhubarb.group
+ * Requires PHP: 7.2
+ */
+
+defined('ABSPATH') || exit;
+
+add_action('muplugins_loaded', ['k6ObjectCacheMetrics', 'init']);
+add_action('shutdown', ['k6ObjectCacheMetrics', 'print'], PHP_INT_MAX);
+
+class k6ObjectCacheMetrics
+{
+    const None = 'none';
+
+    const ObjectCachePro = 'object-cache-pro';
+
+    const RedisObjectCache = 'redis-cache';
+
+    const WpRedis = 'wp-redis';
+
+    const LiteSpeedCache = 'litespeed-cache';
+
+    protected static $cache;
+
+    protected static $client;
+
+    public static function init(): void
+    {
+        global $wp_object_cache;
+
+        if (! wp_using_ext_object_cache()) {
+            self::$cache = self::None;
+            self::$client = 'array';
+        }
+
+        $class = get_class($wp_object_cache);
+
+        if (strpos($class, 'RedisCachePro') === 0) {
+            self::$cache = self::ObjectCachePro;
+            self::$client = strtolower($wp_object_cache->clientName());
+        }
+
+        if (method_exists($class, 'connect_using_relay')) {
+            self::$cache = self::RedisObjectCache;
+
+            $client = class_exists('Redis') ? 'phpredis' : 'predis';
+            self::$client = defined('WP_REDIS_CLIENT') ? str_replace('pecl', 'phpredis', WP_REDIS_CLIENT) : $client;
+        }
+
+        if (method_exists($class, '_connect_redis')) {
+            self::$cache = self::WpRedis;
+            self::$client = 'phpredis';
+        }
+
+        if (property_exists($class, '_object_cache')) {
+            self::$cache = self::LiteSpeedCache;
+            self::$client = 'phpredis';
+        }
+    }
+
+    public static function print(): void
+    {
+        if (! self::$cache) {
+            return;
+        }
+
+        if (! self::shouldPrint()) {
+            return;
+        }
+
+        printf(
+            "\n<!-- plugin=%s client=%s %s -->\n",
+            self::$cache,
+            self::$client,
+            self::getMetrics()
+        );
+    }
+
+    protected static function shouldPrint(): bool
```

Comment:
Let's add the code 1:1 from Object Cache Pro here that's in `incompatibleContentType()` to avoid issues.